include/range/v3/range/access.hpp

```diff
@@ -31,11 +31,20 @@
 #include <range/v3/iterator/reverse_iterator.hpp>
 #include <range/v3/iterator/traits.hpp>
 #include <range/v3/utility/static_const.hpp>
+#include <concepts/type_traits.hpp>
 
 #include <range/v3/detail/disable_warnings.hpp>
 
 namespace ranges
 {
+    namespace _safe_
+    {
+        template<typename T>
+        CPP_concept_bool safe =
+                std::is_lvalue_reference<T>::value ||
+                enable_safe_range<concepts::detail::remove_cvref_t<T>>;
```

Comment:
I think I would prefer this:

```c++
template<typename T>
RANGES_INLINE_VAR constexpr bool safe =
    enable_safe_range<concepts::detail::remove_cvref_t<T>>;

template<typename T>
RANGES_INLINE_VAR constexpr bool safe<T &> = true;
```

No need to instantiate `std::is_lvalue_reference`.